tests/Database/DatabaseEloquentModelTest.php

```diff
@@ -1593,6 +1593,10 @@ public function testAppendingOfAttributes()
         $this->assertSame('camelCased', $model->camelCased);
         $this->assertSame('StudlyCased', $model->StudlyCased);
 
+        $this->assertTrue($model->hasAppended('is_admin'));
+        $this->assertTrue($model->hasAppended('camelCased'));
+        $this->assertTrue($model->hasAppended('StudlyCased'));
```

Comment:
Can we also test the unhappy path here?

Reply:
[Sure can!](https://github.com/laravel/framework/pull/33422/commits/17a14d998910e6889699a4feb7da52b39bd04d0b)

^ Is that alright? Or would you prefer an existing accessor?

Reply:
Perfect : )